src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import format from 'date-fns/format';
+
+import './DashboardBriefs.css'
+
+
+export const DashboardBriefs = ({live, closed, draft, all}) => (
+  <div className="content-main">
+    <DraftBriefs draft={draft}/>
+    <div>
+      <p>
+        <a href={(!isEmpty(all) ?
+          "/buyers/frameworks/".concat(
+            all[0].frameworkSlug,
```

Comment:
This probably won't work for new teams without any "all" briefs or old teams who have only had DSP framework briefs but should now only make DM framework briefs?